DigitalLearningSolutions.Web/Controllers/LearningPortalController/SelfAssessment.cs

```diff
@@ -1355,6 +1356,7 @@ public IActionResult WithdrawSupervisorVerificationRequest(int selfAssessmentId,
         [HttpPost]
         public IActionResult SubmitVerification()
         {
+            var delegateUserId = User.GetUserIdKnownNotNull();

```

Comment:
remove this line since this is an unnecessary change. In this method, you have introduced some line spaces as well which are not required.